internal/pkg/service/stream/definition/repository/branch_repo_test.go

```diff
@@ -1,380 +0,0 @@
-package repository_test
-
-import (
-	"context"
-	"net/http"
-	"testing"
-
-	"github.com/benbjohnson/clock"
-	"github.com/keboola/go-client/pkg/keboola"
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-
-	deps "github.com/keboola/keboola-as-code/internal/pkg/service/common/dependencies"
-	serviceErrors "github.com/keboola/keboola-as-code/internal/pkg/service/common/errors"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/common/utctime"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/definition"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/definition/key"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/dependencies"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/storage/test"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/etcdhelper"
-)
-
-func TestRepository_Branch(t *testing.T) {
-	t.Parallel()
```

Comment:
Old big test is replaced by a separated test for each repository operation, see next commits.